src/background/index.ts

```diff
@@ -253,6 +255,21 @@ browser.runtime.onMessage.addListener(
     openDappletHome: overlayService.openDappletHome.bind(overlayService),
     openDappletAction: overlayService.openDappletAction.bind(overlayService),
     openPopupOverlay: overlayService.openPopupOverlay.bind(overlayService),
+    openConnectedAccountsPopup: overlayService.openConnectedAccountsPopup.bind(overlayService),
+
+    // Connected Account Service
+    getConnectedAccounts:
+      connectedAccountService.getConnectedAccounts.bind(connectedAccountService),
+    getMinStakeAmount: connectedAccountService.getMinStakeAmount.bind(connectedAccountService),
+    getPendingRequests: connectedAccountService.getPendingRequests.bind(connectedAccountService),
+    getVerificationRequest:
+      connectedAccountService.getVerificationRequest.bind(connectedAccountService),
+    getStatus: connectedAccountService.getStatus.bind(connectedAccountService),
+    getMainAccount: connectedAccountService.getMainAccount.bind(connectedAccountService),
+    getRequestStatus: connectedAccountService.getRequestStatus.bind(connectedAccountService),
+    requestVerification: connectedAccountService.requestVerification.bind(connectedAccountService),
```

Comment:
I think it will be clearer 😁
```typescript
    // Connected Account Service
    getConnectedAccounts:
      connectedAccountService.getConnectedAccounts.bind(connectedAccountService),
    getConnectedAccountsMinStakeAmount:
      connectedAccountService.getMinStakeAmount.bind(connectedAccountService),
    getConnectedAccountsPendingRequests:
      connectedAccountService.getPendingRequests.bind(connectedAccountService),
    getConnectedAccountsVerificationRequest:
      connectedAccountService.getVerificationRequest.bind(connectedAccountService),
    getConnectedAccountStatus: connectedAccountService.getStatus.bind(connectedAccountService),
    getConnectedAccountsMainAccount:
      connectedAccountService.getMainAccount.bind(connectedAccountService),
    getConnectedAccountsRequestStatus:
      connectedAccountService.getRequestStatus.bind(connectedAccountService),
    requestConnectingAccountsVerification:
      connectedAccountService.requestVerification.bind(connectedAccountService),
    changeConnectedAccountStatus:
      connectedAccountService.changeStatus.bind(connectedAccountService),
    getConnectedAccountsPairs: connectedAccountService.getPairs.bind(connectedAccountService),
```

Reply:
The methods' names are changed)